### DISCUSSION TITLE
Prevent RichText editor to shorten URLs pointing to same origin

### DISCUSSION THREAD

**Discussion by apikee** [pocketbase/pocketbase#3155] [Q&A]
Hello, I would like to store email templates in column with RichText type. The problem is that when I insert an image into the editor (whether via UI or HTML) that is located on the same origin as PocketBase, the editor truncates the origin to two dots. I.e. from 
```
<img src="http://localhost:8090/track/1234/5678">
```
to
```
<img src="../track/1234/5678">
```
I tried to do the same on the Demo address (pocketbase.io/demo) and the same thing happens there. Is it possible to prevent this? Thanks

**Comment by ganigeorgiev** [comment]
This is the default behavior of TinyMCE (see https://www.tiny.cloud/docs/tinymce/6/url-handling/#relative_urls).

I've added it to the [roadmap](https://github.com/orgs/pocketbase/projects/2/views/1) to consider making it configurable as field option for the next release as it was requested several times already.
